tornado/ioloop.py

```diff
@@ -714,7 +714,8 @@ def run_in_executor(
                 from tornado.process import cpu_count
 
                 self._executor = concurrent.futures.ThreadPoolExecutor(
-                    max_workers=(cpu_count() * 5)
+                    max_workers=(cpu_count() * 5),
+                    thread_name_prefix='default-tornado-executor'
```

Comment:
I'm not sure what the long term plans are for Python version support, but currently we support 3.5.2 and higher. `thread_name_prefix` was added in Python 3.6.

Reply:
We've already dropped support for Python 3.5 in the main branch, so Tornado 6.2 will require python 3.6+ and this change is fine in that respect. (and it looks like we've got less than 6 months before 3.6 reaches EOL so depending on when I get around to making a release, we may drop 3.6 as well)